tests/foreman/smoke/test_api_smoke.py

```diff
@@ -608,6 +611,19 @@
 }
 
 
+def _bz_1184170_affects_server():
```

Comment:
If we see another issue needing this, then we create. For this, a "private" function is good.